src/Workspaces/Remote/ServiceHub.CoreComponents/CoreComponents.Shared.targets

```diff
@@ -19,8 +19,6 @@
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\ServiceHub\Microsoft.CodeAnalysis.Remote.ServiceHub.csproj" />
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\..\..\Features\CSharp\Portable\Microsoft.CodeAnalysis.CSharp.Features.csproj" />
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\..\..\Features\VisualBasic\Portable\Microsoft.CodeAnalysis.VisualBasic.Features.vbproj" />
-    <!-- CodeLens provider is not part of Roslyn ServiceHub, we just use the existing ServiceHub package as a delivery vehicle -->
```

Comment:
So was this comment wrong that we weren't using it as a delivery vehicle? Or does this need to be undone with the other CodeLens changes?

Reply:
yeah.  this needs to be undone.  it broke VS insertion.  But i don't understand why.   It seems bad that we depend on this line here for things to work.  i would love your help on understanding that @JoeRobich @jasonmalinowski @ToddGrun .